.github/workflows/slither-analysis.yml

```diff
@@ -1,11 +1,7 @@
 name: Slither Analysis
 
-on:
-  push:
-    branches:
-      - development
-      - main
-  pull_request:
+on: [push, pull_request]
```

Comment:
Why was it updated?

Reply:
@AnakinSkywalkeer can put some wording here?

Reply:
this is already fixed